src/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+Adding the following methods:
```

Comment:
Can we just put this info in the PR descriptions?
Since we don't have this log before, let's use the PR description to track the changes

Reply:
Done